src/components/Login/components/authError/AuthError.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import styles from './AuthError.css';
+
+export default class AuthError extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Please add isRequired to errors property or defaultProps if this property is not required